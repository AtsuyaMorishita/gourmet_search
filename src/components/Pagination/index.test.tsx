import { render } from "@testing-library/react";
import "@testing-library/jest-dom";
import { Pagination } from ".";

describe("Pagination", () => {
  it("render correctly", () => {
    const { container } = render(<Pagination currentPage={1} path="/search" />);
    expect(container).toMatchSnapshot();
  });
});
