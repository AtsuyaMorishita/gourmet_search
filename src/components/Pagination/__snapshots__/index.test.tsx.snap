// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pagination render correctly 1`] = `
<div>
  <section>
    <ul
      class="flex justify-center"
    >
      <li />
      <li />
    </ul>
  </section>
</div>
`;
